go/grpcweb/wrapper.go

```diff
@@ -158,13 +164,19 @@ func (w *WrappedGrpcServer) handleWebSocket(wsConn *websocket.Conn, req *http.Re
 	req.Method = http.MethodPost
 	req.Header = headers
 
-	w.server.ServeHTTP(respWriter, hackIntoNormalGrpcRequest(req.WithContext(ctx)))
+	// TODO: pass isTextFormat into newWebSocketResponseWriter and figure out how that works
```

Comment:
Can we add an issue to track the work on this and reference it here, if it's not going in at the same time?

Reply:
I created #294 to record this. I've removed this TODO.